perl/dist.ini

```diff
@@ -0,0 +1,48 @@
+name      = Gherkin
+abstract  = A parser and compiler for the Gherkin language
+main_module = lib/Gherkin.pm
+author    = ['Peter Sergeant <pete@clueball.com>', 'Gaspar Nagy']
```

Comment:
Please add `Cucumber Ltd` as one of the authors - should be same as in `/LICENSE`